index.html

```diff
@@ -0,0 +1,167 @@
+            </ul>
+          </div>
+        </div>
+      </nav>
+      <div class="landing">
+        <div class="window">
+          <ul class="window-list">
+            <li>
+              <a href="./about.html" target="_blank" class="menu-link">About</a>
+            </li>
+            <li><a href="#main-program" class="menu-link">Program</a></li>
+            <li><a href="#featured-speakers" class="menu-link">Speakers</a></li>
+          </ul>
+        </div>
+
+        <h2 class="introh2">"Hello! caring African world"</h2>
+        <div class="h1div">
+          <h1 class="introh1">EAST AFRICAN GLOBAL WARMING SUMMIT 2023</h1>
+        </div>
+        <p class="introp">
+          Coming together of three of the most affected nations within the
+          Eastern and Central African countries to explore and understand the
+          socio-economic effects of global warming to the nations.
+        </p>
+        <div class="date">
+          <p><span>2023.03.15</span>(WED) ~ <span>20</span>(SAT)</p>
+        </div>
+        <p class="location">@Kitengela, Naivash Shopping Center, Nairobi</p>
+      </div>
+    </header>
+    <section class="main-program" id="main-program">
+      <h1>Main Program</h1>
+      <div class="underline"></div>
+      <div class="program-container">
+        <div class="card">
+          <img src="./assets/lecture.png" alt="lectur icon" />
+          <h2>Livetalks</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, sint!
+          </p>
+        </div>
+        <div class="card">
+          <img src="./assets/networking.png" alt="networking icon" />
+          <h2>Network</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, sint!
+          </p>
+        </div>
+        <div class="card">
+          <img src="./assets/group (2).png" alt="community icon" />
+          <h2>Forum</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, sint!
+          </p>
+        </div>
+        <div class="card">
+          <img src="./assets/visitors.png" alt="vspeakers icon" />
+          <h2>Showroom</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, sint!
+          </p>
+        </div>
+        <div class="card">
+          <img src="./assets/workshop.png" alt="workshop icon" />
+          <h2>Workshop</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, sint!
+          </p>
+        </div>
+      </div>
+      <button type="submit" class="join-button">
+        Join ECA African Summit 2023
+      </button>
+      <p class="see-prog">SEE THE WHOLE PROGRAM</p>
+    </section>
+    <h1 class="featured-h1">Featured Speakers</h1>
+    <div class="underline spacing"></div>
+    <section class="featured-speakers" id="featured-speakers">
+      <!-- content load dynamically  -->
+    </section>
```

Comment:
- [ ] In your mobile version, the  `featured speakers` section is missing the `more` button. Kindly add it to match the original design

![image](https://user-images.githubusercontent.com/98453979/217302536-edfa65e0-23c3-484d-9e18-78d8cdddbded.png)